PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)
+
+[//]: # (Put Issue # here, if applicable)
+Fixes #______
```

Comment:
No, just the URL, not the number at all. Either will cause a link to be made to the issue, and when the PR is saved, the URL will look like the number.